### PR TITLE
Added RO and BG

### PR DESCRIPTION
**Pull request by raresvanca**
Added support for RO(romanian) and BG(bulgarian)

### CODE REVIEW

**Comment by Lukewh** [comment]
Thanks for the contribution - sorry for missing it. Can you pass the SVGs through SVGOMG to remove the Inkscape/  unnecessary metadata please?
There's an online version here: https://jakearchibald.github.io/svgomg/

Thanks

**Comment by lilyvidenova** [comment]
+1 for the Bulgarian version, thanks@rares45! :)

**Comment by solazio** [comment]
+1 for Romanian. Thanks @rares45

File: RO/[RO]-snap-store-black.svg [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   id="body" />
+			<polygon
+   points="29.9,6.8 19.4,6.8 32.9,12.9    "
+   class="st2"
+   id="head" />
+		</g>
+		<text
+   id="text12"
+   class="st3 st4 st5"
+   transform="translate(50,17)">Instalează-l din</text>
```
Comment (solazio) [inline RO/[RO]-snap-store-black.svg]:
```suggestion
   transform="translate(50,17)">Instalează din</text>
```

File: RO/[RO]-snap-store-white.svg [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   id="body" />
+			<polygon
+   points="29.9,6.8 19.4,6.8 32.9,12.9    "
+   class="st2"
+   id="head" />
+		</g>
+		<text
+   id="text12"
+   class="st3 st4 st5"
+   transform="translate(50,17)">Instalează-l din</text>
```
Comment (solazio) [inline RO/[RO]-snap-store-white.svg]:
```suggestion
   transform="translate(50,17)">Instalează din</text>
```

**Comment by raresvanca** [comment]
@Lukewh
> Can you pass the SVGs through SVGOMG to remove the Inkscape/ unnecessary metadata please?

Working on that

**Review by Lukewh — APPROVED** [review]
:+1: Looks good, thanks @rares45

**Comment by raresvanca** [comment]
@Lukewh @solazio what about merging?

**Comment by Lukewh** [comment]
@rares45 I've created this issue: https://github.com/canonical-web-and-design/snapcraft.io/issues/2839 to track adding that. It's the second thing for me to do today :).

**Comment by Lukewh** [comment]
@rares45 I'm also trying to get permission from the right people to merge this - forgot to say that :)

**Pull request merged**